### PR TITLE
Add docstring to leapp.actors

### PR DESCRIPTION
**Pull request by jzigmund**

### CODE REVIEW

**Comment by jzigmund** [comment]
@vinzenz added 2 commits to the PR

File: docs/source/conf.py [hunk 1]
```diff
@@ -171,24 +173,21 @@
...
     if name.startswith('with_meta_base_') or name == 'mro':
         return True
     if name == 'commands' and what == 'module':
         return True
     return skip
 
-
 def setup(app):
+
+    # prefix each section label with the name of the document it is in, followed by a colon
```
Comment (vinzenz) [inline docs/source/conf.py]: This comment should be removed now, that we do our own plugin instead

**Comment by jzigmund** [comment]
@vinzenz fixed minor issues after your review, updated PR.

File: leapp/actors/__init__.py [hunk 2]
```diff
@@ -13,73 +13,130 @@
...
             os.environ.pop('LEAPP_CURRENT_ACTOR', None)
 
     def process(self, *args, **kwargs):
+        """ Main processing method (in inherited actors, the function needs to be defined to be able process)"""
         raise NotImplementedError()
 
-    def produce(self, *args):
+    def produce(self, *models):
+        """
+        After running :py:func:`process`, it prepares processed data - results to next processing in workflow.
```
Comment (vinzenz) [inline leapp/actors/__init__.py]: No this is not correct it is immediately when produce is called.
User rather this:

By calling produce model instances are store as messages. Those messages can be then consumed by other actors.

File: leapp/actors/__init__.py [hunk 3]
```diff
@@ -13,73 +13,130 @@
...
+
+        :param models: what data models is going to be produced
```
Comment (vinzenz) [inline leapp/actors/__init__.py]: :param models: Messages to be sent (those model types have to be specified in :py:attr:`~produces`
:type models: Variable number of :py:class:`leapp.models.Model` derived classes

File: leapp/actors/__init__.py [hunk 4]
```diff
@@ -13,73 +13,130 @@
...
                         'message': {
                             'data': message_data,
                             'hash': message_hash
                         }
                     })
 
-    def consume(self, *types):
+    def consume(self, *models):
+        """
+        Load any given model data types to be able to process them.
```
Comment (vinzenz) [inline leapp/actors/__init__.py]: Retrieve messages specified in the actors :py:attr:`~consumes` attribute and can be filter the message types by `models`

File: leapp/actors/__init__.py [hunk 5]
```diff
@@ -13,73 +13,130 @@
...
+
+        :param models: what data models is going to be processed
+        :type models: tuple of derived classes from :py:class:`leapp.models.Model`
```
Comment (vinzenz) [inline leapp/actors/__init__.py]: var arg not really tuple even though it looks like a tuple in the code

File: leapp/actors/__init__.py [hunk 6]
```diff
@@ -13,73 +13,130 @@
...
                             'hash': message_hash
                         }
                     })
 
-    def consume(self, *types):
+    def consume(self, *models):
+        """
+        Load any given model data types to be able to process them.
+
+        :param models: what data models is going to be processed
```
Comment (vinzenz) [inline leapp/actors/__init__.py]: Models to use as filter for the messages to return

**Pull request merged**